zerver/lib/users.py

```diff
@@ -548,6 +671,74 @@ def get_cross_realm_dicts() -> List[APIUserDict]:
     return result
 
 
+def get_data_for_inaccessible_user(realm: Realm, user_id: int) -> APIUserDict:
+    user_email = Address(username=f"user{user_id}", domain=get_fake_email_domain(realm)).addr_spec
```

Comment:
Renamed this variable to `fake_email` to make it more obvious we're not sending a real email here.